auth/src/main/java/com/firebase/ui/auth/ui/phone/PhoneVerificationActivity.java

```diff
@@ -386,7 +383,7 @@ private void showLoadingDialog(String message) {
 
     private void dismissLoadingDialog() {
         if (mProgressDialog != null) {
-            mProgressDialog.dismiss();
+            mProgressDialog.dismissAllowingStateLoss();
```

Comment:
Santa Tracker was started as a holiday side project by someone on Maps Developer Relations, so it's stayed in the DevRel family.  It's a good way for us to remember what it's like to make a web/android app in the real world, and at this point it's a Google holiday tradition.